src/control/cmd/dmg/storage_query.go

```diff
@@ -142,16 +156,33 @@ func (cmd *usageQueryCmd) Execute(_ []string) error {
 		return err
 	}
 
-	var bld strings.Builder
-	if err := pretty.PrintResponseErrors(resp, &bld); err != nil {
+	var outErr strings.Builder
+	if err := pretty.PrintResponseErrors(resp, &outErr); err != nil {
 		return err
 	}
-	if err := pretty.PrintHostStorageUsageMap(resp.HostStorage, &bld); err != nil {
-		return err
+	if outErr.Len() > 0 {
+		cmd.Error(outErr.String())
+	}
+
+	var out, dbg strings.Builder
+	if resp.HostStorage.IsMdOnSsdEnabled() {
+		if err := pretty.PrintHostStorageUsageMapMdOnSsd(resp.HostStorage, &out, &dbg, cmd.ShowUsable); err != nil {
+			cmd.Error(err.Error())
+		}
+	} else {
+		if cmd.ShowUsable {
+			cmd.Notice("--show-usable flag ignored when MD-on-SSD is not enabled")
+		}
+		pretty.PrintHostStorageUsageMap(resp.HostStorage, &out)
+	}
+	if dbg.Len() > 0 {
+		cmd.Debugf("%s", dbg.String())
```

Comment:
```suggestion
		cmd.Debug(dbg.String())
```

Reply:
Ah, the newlines aren't parsed when you just print the string? Didn't know that. That sort of seems like a bug in our logging to be honest.

Reply:
actually I think it was something to do with the percentage sign parsing

Reply:
Oh, hm. That makes some sense, but it seems like the non-format version shouldn't be mangling anything at all.